revocation/result/results.go

```diff
@@ -90,14 +98,28 @@ type CertRevocationResult struct {
 	// Thus, in the case of more than one ServerResult, this will be ResultUnknown
 	Result Result
 
-	// An array of results for each server associated with the certificate.
-	// The length will be either 1 or the number of OCSPServers for the cert.
+	// ServerResults is an array of results for each server associated with the
+	// certificate.
```

Comment:
Comments for `Result Result` at line 94 to 98 is not accurate anymore since the length of `ServerResult` can be greater than one due to concatenation of OCSP and CRL results.

Reply:
Removed.